rollingpin/harold.py

```diff
@@ -13,6 +13,8 @@
 
 from .utils import swallow_exceptions
 
+TIMEOUT_SECONDS = 5
```

Comment:
lmk if you think this should be configurable from the .ini

Reply:
meh, it's not gonna change in any other environments so it's fine.